tests/splines/CMakeLists.txt

```diff
@@ -101,18 +101,38 @@ foreach(BC "BC_PERIODIC" "BC_GREVILLE")
   endforeach()
 endforeach()
 
+# LAPACK
 foreach(BC "BC_PERIODIC" "BC_GREVILLE" "BC_HERMITE")
   foreach(DEGREE_X RANGE "${SPLINES_TEST_DEGREE_MIN}" "${SPLINES_TEST_DEGREE_MAX}")
     foreach(BSPLINES_TYPE "BSPLINES_TYPE_UNIFORM" "BSPLINES_TYPE_NON_UNIFORM")
-      set(test_name "splines_tests_BATCHED_DEGREE_X_${DEGREE_X}_${BSPLINES_TYPE}_${BC}")
+      set(test_name "splines_tests_BATCHED_DEGREE_X_${DEGREE_X}_${BSPLINES_TYPE}_${BC}_LAPACK")
       add_executable("${test_name}" ../main.cpp batched_spline_builder.cpp)
       target_compile_features("${test_name}" PUBLIC cxx_std_17)
       target_link_libraries("${test_name}"
         PUBLIC
         GTest::gtest
         DDC::DDC
       )
-      target_compile_definitions("${test_name}" PUBLIC -DDEGREE_X=${DEGREE_X} -D${BSPLINES_TYPE} -D${BC})
+    target_compile_definitions("${test_name}" PUBLIC -DDEGREE_X=${DEGREE_X} -D${BSPLINES_TYPE} -D${BC} -DSOLVER_LAPACK)
+      # add_test("${test_name}" "${test_name}")
+      gtest_discover_tests("${test_name}" DISCOVERY_MODE PRE_TEST)
+    endforeach()
+  endforeach()
+endforeach()
+
+# GINKGO
+foreach(BC "BC_PERIODIC")
+  foreach(DEGREE_X RANGE "${SPLINES_TEST_DEGREE_MIN}" "${SPLINES_TEST_DEGREE_MAX}")
+    foreach(BSPLINES_TYPE "BSPLINES_TYPE_UNIFORM")
```

Comment:
The scope of GINKGO backend tests is reduced (only PERIODIC/UNIFORM is tested) to reduce the number of tests.

Reply:
I was thinking of removing C++23 tests, this would mitigate the cost of adding the LAPACK tests

Reply:
As you want, please tell me what to do with tests.

Reply:
Keep the Ginkgo tests as in the main branch for now